npm/private/test/package.json

```diff
@@ -2,7 +2,7 @@
     "private": true,
     "// devDependencies": "test linking & building 3rd party npm packages with problematic postinstall",
     "devDependencies": {
-        "@blockprotocol/type-system-web": "https://gitpkg.vercel.app/blockprotocol/blockprotocol/packages/%40blockprotocol/type-system-web?6526c0e",
+        "hello": "https://gitpkg.vercel.app/EqualMa/gitpkg-hello/packages/hello",
```

Comment:
What is this?

Reply:
Related to https://github.com/pnpm/pnpm/issues/5414. I'll pre-factor this and self-contained changes out.

Reply:
Thanks for pulling those out 👍